### DISCUSSION TITLE
Pledge Works feedback

### DISCUSSION THREAD

**Discussion by ialja** [ResponsibleTechWork/RespTechWork-website#14] [Pledge Works]
Got any feedback on [Pledge Works](https://responsibletech.work/tools/development/pledge-works/)? We'd love to hear from you! :)

**Comment by m3data** [comment]
Firstly just fantastic work! Really like the nesting of purpose, values, principles and pledges in the ethical solar system and the ethics tree in the toolbox 💜 

Very excited to use it at Tethix and contribute to it's evolution 🌱
